rotkehlchen/tests/api/blockchain/test_evm.py

```diff
@@ -311,6 +311,7 @@ def test_add_multievm_accounts(rotkehlchen_api_server):
     ]
 
 
+@pytest.mark.skip()  # TODO: remove when basescan is back
```

Comment:
Whenever you do such a thing please make sure there is a note here: https://github.com/orgs/rotki/projects/11 otherwise this can very easily be forgotten forever

Reply:
Acknowledged, thanks!